config/environments/development.rb

```diff
@@ -59,6 +59,7 @@
 
   config.hosts << "lvh.me"
   config.hosts << "website" if ENV['EXERCISM_DOCKER']
+  config.hosts << /.*.ngrok.io/
```

Comment:
Is this an alternative to lvh.me?

Reply:
No, [ngrok](https://ngrok.com/) is a service that allows you to expose your locahost as a "real" URL. 

So I run:
```
ngrok http 3020
```

and it gives me an ngrok URL (`abc123def456.ngrok.io`), which I can then give to you, and you can then access my localhost:3020.

It's very useful. You can `brew install` in on all reasonable OSs :wink: